apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/SkeletonRDASink.java

```diff
@@ -0,0 +1,31 @@
+package gov.cms.bfd.pipeline.rda.grpc;
+
+import com.codahale.metrics.Meter;
+import com.codahale.metrics.MetricRegistry;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Skeleton RDASink implementation that just updates a Meter to count the number of times it has
+ * been called. This implementation satisfies the requirements of DCGEO-18.
+ */
+public class SkeletonRDASink implements RDASink<PreAdjudicatedClaim> {
```

Comment:
Come implementation time do you think you'll remove the prefix `Skeleton`?

Reply:
Yes this class would be replaced with the real sink.  That sink will have an appropriate name like PostgresRDASink.